source/examples/cubescape-log/CubeScape.cpp

```diff
@@ -0,0 +1,227 @@
+
```

Comment:
I think it should be possible to reuse the files from the ```cubescape``` example as it is done by the ```cubescape-qt``` example.
Please verify this and adjust the code accordingly.
I don't want to have two implementations of the ```CubeScape``` and related functionality.